test/common/config.js

```diff
@@ -14,6 +16,22 @@ export default function () {
       kibana: kibanaTestServerUrlParts,
       elasticsearch: esTestConfig.getUrlParts(),
     },
+
+    kibanaServerArgs: [
+      '--env=development',
+      '--logging.json=false',
+      '--no-base-path',
+      `--server.port=${kibanaTestServerUrlParts.port}`,
+      `--optimize.watchPort=${kibanaTestServerUrlParts.port}`,
+      '--optimize.watchPrebuild=true',
+      '--status.allowAnonymous=true',
+      '--optimize.enabled=true',
+      `--optimize.bundleDir=${OPTIMIZE_BUNDLE_DIR}`,
+      `--elasticsearch.url=${formatUrl(esTestConfig.getUrlParts())}`,
```

Comment:
__optional nit:__ hmm, maybe let's just move `esTestConfig.getUrlParts()` to a variable (it's called 4 times in this function)?

Reply:
or rather, use the existing `servers.elasticsearch` definition in this file. ;)